via/configuration.py

```diff
@@ -107,12 +107,3 @@ def _move_legacy_params(via_params, client_params):
 
         # This is like to be around for a while
```

Comment:
"This is like to be around for a while" - old comment, but should this read "likely" ?